migrations/migrations/20210406205811-name-of-migration.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+      return queryInterface.createTable('user', {
+        id: {
+          type: Sequelize.INTEGER,
+          unique: true,
+          autoIncrement: true,
+          primaryKey: true,
+          allowNull: false
+        },
+        firstName: {
```

Comment:
migration attributes is not camelCase are `snake_case`

Reply:
column name fixed